man/org.freedesktop.systemd1.xml

```diff
@@ -1486,6 +1486,11 @@ node /org/freedesktop/systemd1 {
       boot loader or initrd implementation. In these cases the respective pairs of timestamps are both 0,
       indicating that no data is available.</para>
 
+      <para><varname>UnitsReloadStartTimestamp</varname> encode <constant>CLOCK_REALTIME</constant>
```

Comment:
hmm, maybe call this UnitLoadTimestamp

i.e. right after boot we never "reloaded", we just "loaded", hence the name should be generic enough to mention that.